roofit/roofit/inc/LinkDef1.h

```diff
@@ -131,10 +131,4 @@
 #pragma link C++ class RooCFunction4Binding<double,double,double,double,bool>+ ;
 #pragma link C++ class RooCFunction4PdfBinding<double,double,double,double,bool>+ ;
 
-//#ifdef USE_FOR_AUTLOADING
-//#pragma link C++ class RooFit ;
-//#else
-#pragma link C++ namespace RooFit ;
```

Comment:
This was the only active one.  It may still be useful for build without the `runtime_cxxmodule`.

Reply:
the modules-off build worked. There are other failures though.

Reply:
Ah thanks for noticing this oversight